examples/client-sdks/react-chat-app.tsx

```
@@ -0,0 +1,1113 @@
+        <div className="empty-state">
+          <p>No tools available</p>
+        </div>
+      )}
+
+      <div className="tools-grid">
+        {tools.map((tool) => (
+          <div
+            key={tool.name}
+            className={`tool-card ${selectedTool === tool.name ? "selected" : ""}`}
+            onClick={() => setSelectedTool(tool.name)}
+          >
```

Biome (2.4.7)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)